### DISCUSSION TITLE
Why No Documentation Feedback?

### DISCUSSION THREAD

**Discussion by ddr5292** [laravel/framework#46709] [General]
I often find that the documentation is unclear but there is no way to provide feedback or ask questions. Normally, there is a mechanism for this in most (MVC provides a feedback button - for example.) Some of these questions are pressing and the lack of clarity prevents us from fully adopting the technology and I have seen people walk away from Laravel as a result. 

It is a huge mistake. I see many questions are answered in Laracasts but the documentation authors do not seem to update the doc  based on the confusion evident in that forum - so there seems to be a missed opportunity here.

**Comment by Sophist-UK** [comment]
Laracasts is NOT Taylor Otwell's Laravel organisation - why would you think that discussions on Laracasts forums would automagically result in documentation improvements?

I agree that (like many other open source projects) there is always scope for documentation improvements. (As an aside, and as a contributor to many open source projects, I can say from personal experience that developers do not make great documentation writers - partly because they are developers, partly because they are often too close to the coalface, and partly because they are not inexperienced users struggling to use the technology for their own unique project.) That said, the Laravel documentation is significantly better than many other open source projects, so please let's not be too hard on @taylorotwell and his team who IMO do a great job of driving Laravel to greater heights and of curating the documentation.

The Laravel documentation repo is [here](https://github.com/laravel/docs). Like almost all open source projects, anyone can submit improvements to this repo via PRs, so please feel free to submit such PRs for documentation improvements you would like to see (based on Laracast discussions) and to encourage others to do the same., 😃
